src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -101,3 +101,647 @@ It includes a uniform request, successful and error response formats. At the mom
+| gas\_price      | numeric string | `Eth` or `Qrc20` type only. Price per unit of gas to be used for the transaction. |
+| gas\_limit      | numeric string | `Tendermint` or `Qrc20` type only. Maximum gas to be used for the transaction.    |
+| miner\_fee      | numeric string | `Tendermint` type only. Fee to mine the transaction.                              |
+| total\_fee      | numeric string | `Eth` type only. Gas price multiplied by gas amount.                              |
+| total\_gas\_fee | numeric string | `Qrc20` type only. Gas price multiplied by gas amount.                            |
+
+<CollapsibleSection expandedText="Hide Examples" collapsedText="Show Examples">
+  #### Example of Eth type
+
+  ```json
+  {
+    "type": "Eth",
+    "coin": "BNB",
+    "gas": 109739,
+    "gas_price": "0.000000003",
+    "total_fee": "0.000329217"
+  }
+  ```
+
+  #### Example of Qrc20 type
+
+  ```json
+  {
+    "type": "Qrc20",
+    "coin": "tQTUM",
+    "miner_fee": "0.00000447",
+    "gas_limit": 100000,
+    "gas_price": 40,
+    "total_gas_fee": "0.04"
+  }
+  ```
+
+  #### Example of Tendermint type
+
+  ```json
+  {
+    "type": "Tendermint",
+    "coin": "IRIS",
+    "amount": "0.038553",
+    "gas_limit": 100000
+  }
+  ```
+
+  #### Example of Utxo type
+
+  ```json
+  {
+    "type": "Utxo",
+    "amount": "0.00001"
+  }
+  ```
+</CollapsibleSection>
+
+### WithdrawNftData
+
+The `WithdrawNftData` object is used for withdrawals of NFTs on ERC721 and ERC1155 contracts. It includes the following items for a given coin or token:
+
+| Parameter      | Type    | Description                                                                                                                              |
+| -------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| chain          | string  | The token chain. Chain must be [activated](/atomicdex/api/legacy/coin_activation/) first.                                                |
+| to             | string  | Destination address to withdraw the token to.                                                                                            |
+| token\_address | string  | Token address.                                                                                                                           |
+| token\_id      | string  | Token ID.                                                                                                                                |
+| fee            | string  | Optional. A standard [WithdrawFee](/atomicdex/api/v20/#withdraw-fee) object.                                                             |
+| amount         | string  | Optional, ERC1155 only. Defaults to `1`. Amount of NFTs to withdraw. Ignored if `max` is \`true.                                         |
```

Comment:
Should `true` be written like this \ `true in docs ?

Reply:
that's a typo. should be `true` cc: @smk762

Reply:
fixed